core/src/main/java/org/bitcoinj/wallet/WalletProtobufSerializer.java

```diff
@@ -281,6 +282,13 @@ public Protos.Wallet walletToProto(Wallet wallet) {
             extendedKeyChainBuilder.addAllKey(wallet.getProviderOperatorKeyChain().serializeToProtobuf());
             extendedKeyChains.add(extendedKeyChainBuilder.build());
         }
+        if(wallet.getProviderPlatformOperatorKeyChain() != null) {
+            Protos.ExtendedKeyChain.Builder extendedKeyChainBuilder = Protos.ExtendedKeyChain.newBuilder();
+            extendedKeyChainBuilder.setKeyType(Protos.ExtendedKeyChain.KeyType.EDDSA);
+            extendedKeyChainBuilder.setType(Protos.ExtendedKeyChain.ExtendedKeyChainType.MASTERNODE_PLATFORM_OPERATOR);
+            extendedKeyChainBuilder.addAllKey(wallet.getProviderPlatformOperatorKeyChain().serializeToProtobuf());
+            extendedKeyChains.add(extendedKeyChainBuilder.build());
+        }
```

Comment:
This is a "manual" addition to support the new key type.  

In a separate PR, this serialization will be refactored into the `AuthenticationGroupExtension` class.